hw3/03-react/src/components/Character.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
If I'm not wrong you might be able to get rid of the react import if its not needed. besides that this component looks great.